pkg/server/skydb/oauthinfo.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2015-present Oursky Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package skydb
+
+import (
+	"time"
+)
+
+type TokenResponse    map[string]interface{}
+type ProviderProfile  map[string]interface{}
+
+// OAuthInfo contains 3rd provider information for authentication
+type OAuthInfo struct {
+	UserID           string            `json:"user_id"`
```

Comment:
This `UserID` refers to our `AuthInfo.ID`? I think it worth an extra comment since `user_id` is ambiguous in this context. It can be the user_id from oauth or out user_id.